### DISCUSSION TITLE
Editing bundled file comment text.

### DISCUSSION THREAD

**Discussion by tabeth** [webpack/webpack#12401] [Q&A] [answered]
When you have a bundled file that begins like so:

```javascript
(function(e, a) { for(var i in a) e[i] = a[i]; }(this, /******/ (function(modules) { // webpackBootstrap
/******/ 	// The module cache
/******/ 	var installedModules = {};
/******/
/******/ 	// The require function
/******/ 	function __webpack_require__(moduleId) {
/******/
/******/ 		// Check if module is in cache
/******/ 		if(installedModules[moduleId]) {
/******/ 			return installedModules[moduleId].exports;
/******/ 		}
/******/ 		// Create a new module (and put it into the cache)
/******/ 		var module = installedModules[moduleId] = {
/******/ 			i: moduleId,
/******/ 			l: false,
/******/ 			exports: {}
/******/ 		};
```

Is there a best practice to change the text `The require function`? The use case is that I am instrumenting bundled code for code coverage purposes and would like to add to/modify the existing comment so that function is ignored for code coverage purposes. Right now, this is done manually with a script but was wondering if there were alternatives. 

I see that the line in question is pushed [here](https://github.com/webpack/webpack/blob/af52c8f0457b1055098d6b7b545b257404d06c93/lib/javascript/JavascriptModulesPlugin.js#L819) but don't know if there are configuration options surrounding it.

**Comment by alexander-akait** [comment]
Why you tested bundle file?

**Reply by tabeth** [reply]
We're testing the bundled file since that's what ultimately served to the browser. There are also limitations around testing non-bundled code in the browser.

**Reply by alexander-akait** [reply]
Okay, but why don't use e2e tests for this, coverage for bundled file is not good idea

**Reply by tabeth** [reply]
Regardless of whether it is a good idea or not, is it possible to edit the bundled file comment text?

**Reply by alexander-akait** [reply]
No, because it is bad idea, so we don't provide ability to do this

**Comment by tabeth** [comment, marked as answer]
Webpack does not offer the ability to edit bundled file comment text [(see conversation above)](https://github.com/webpack/webpack/discussions/12401#discussioncomment-276713).

**Reply by alexander-akait** [reply]
You can write small script to replace this comments on what you need, it is easy

**Reply by tabeth** [reply]
Yes, this is what we're doing already. We were just wondering if there was anything native that we were missing. Thanks for your prompt responses.
